src/components/Budget/AddBudgetModal/AddBudgetModal.js

```diff
@@ -67,6 +99,12 @@ AddBudgetModal.manifest = Object.freeze({
     fetch: false,
     accumulate: true,
   },
+  currentFY: {
```

Comment:
this manifest name is already used in FundCurrentBudgetContainer

Reply:
Renamed it